App/server.tsx

```diff
@@ -96,12 +96,25 @@ const lock = util.promisify(lockFile.lock);
 const unlock = util.promisify(lockFile.unlock);
 const writeFile = util.promisify(fs.writeFile);
 const readFile = util.promisify(fs.readFile);
+const mkdir = util.promisify(fs.mkdir);
 
-const file = "./config.json";
+const file = "./data/config.json";
 const saveConfiguration = debounce(async () => {
+  console.log("Trying to save configuration");
+  try {
+    await mkdir("./data");
+  } catch (err) {
+    // Ignore if the directory already exist.
```

Comment:
you can also use `fs.exists`

Reply:
`exists` is deprecated: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback

And we instead must use `fs.stat` and see if it throws, so there is not much difference between the two, in the end :smile:

Reply:
I didn't realize there was a difference between `exists` and `existsSync`:
> fs.exists() is deprecated, but fs.existsSync() is not